src/test/groovy/com/tomtom/gradle/updatechangelog/UpdateChangelogPluginTest.groovy

```diff
@@ -0,0 +1,20 @@
+package com.tomtom.gradle.updatechangelog;
+
+import static org.junit.jupiter.api.Assertions.*
+
+import org.junit.jupiter.api.Test
+import org.gradle.api.Project
+import org.gradle.testfixtures.ProjectBuilder
+
+
+public class UpdateChangelogPluginTest {
```

Comment:
Wouldn't a test validating the real flow be nice to add? That is actually the purpose of the plugin.

Reply:
Yes, definitely.